msg/rate_ctrl_status.msg

```diff
@@ -0,0 +1,11 @@
+
+# rates used by the controller
+float32 rollspeed		# Bias corrected angular velocity about X body axis in rad/s
+float32 pitchspeed		# Bias corrected angular velocity about Y body axis in rad/s
+float32 yawspeed		# Bias corrected angular velocity about Z body axis in rad/s
+
+# rate controller integrator status
+float32 roll_integ		# roll rate integrator
```

Comment:
Why not name it `rollrate_integrator` or `rollspeed_integrator`? It would make the comment redundant and the code more readable.

Reply:
It seemed consistent with https://github.com/PX4/Firmware/blob/master/msg/vehicle_rates_setpoint.msg, but I'm good with the more descriptive naming.